utils/lnurl/index.js

```diff
@@ -0,0 +1,89 @@
+import axios from 'axios'
+import bech32 from 'bech32'
+
+/**
+ *uintToString - Converts uint array to string.
+ *
+ * @param {Array} uintArray array
+ * @returns {string} converted string
+ */
+function uintToString(uintArray) {
+  const encodedString = String.fromCharCode.apply(null, uintArray)
+  return decodeURIComponent(escape(encodedString))
+}
+
+/**
+ * fetchWithdrawParams - Fetches withdrawal request params from the specified `lnurl`
+ *
+ * @param {string} lnurl url
+ * @returns {object} request params
+ * Throws exception if target `lnurl` is not withdrawal request of params weren't successfully
+ * fetched
+ */
+export async function fetchWithdrawParams(lnurl) {
+  const {
+    data: {
+      // a second-level url which would accept a withdrawal Lightning invoice as query parameter
+      callback,
+      // an ephemeral secret which would allow user to withdraw funds
+      k1: secret,
+      maxWithdrawable,
+      // A default withdrawal invoice description
+      defaultDescription,
+      // An optional field, defaults to 1 milisats if not present, can not be less than 1 or more than `maxWithdrawable`
+      minWithdrawable,
+      // action type
+      tag,
+    },
+  } = await axios.get(lnurl)
+
+  if (tag === 'withdrawRequest') {
+    return {
+      callback,
+      secret,
+      maxWithdrawable,
+      minWithdrawable,
+      defaultDescription,
+    }
+  }
+
+  throw new Error('Unknown request type')
+}
+
+/**
+ * makeWithdrawRequest - Attempts withdrawal with the service specified by `callback`.
+ *
+ * @export
```

Comment:
We don't use this `@export` tag anywhere else in the app and it fails linting with the warning:

> Invalid JSDoc tag name "export". (jsdoc/check-tag-names)